requirements.txt

```diff
@@ -3,3 +3,5 @@ fs
 mako
 boto
 bleach==2.1.4
+pycodestyle
+pylint
```

Comment:
Probably not worth adding these until we actually run them, and once we do we'd want to use edx-lint instead of pylint directly.

Reply:
Added a run of __pycodestyle__ but removed __pylint__.

Do we acutally _run_ __edx-lint__ somewhere?  I see it in requirements files but I do not see where it is actually run.